cluster-autoscaler/cloudprovider/aws/examples/cluster-autoscaler-autodiscover.yaml

```
@@ -136,9 +136,13 @@ spec:
         prometheus.io/scrape: 'true'
         prometheus.io/port: '8085'
     spec:
+      priorityClassName: system-cluster-critical
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
       serviceAccountName: cluster-autoscaler
       containers:
-        - image: k8s.gcr.io/autoscaling/cluster-autoscaler:v1.17.3
+        - image: k8s.gcr.io/autoscaling/cluster-autoscaler:v1.22.0
```

As this is [linked to](https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html#ca-deploy) by the AWS EKS docs, can we instead bump this to the latest patch version of 1.21 instead as that's the newest minor version currently supported by AWS.

You've also bumped this in 3/4 of the examples, can we bump it in the last of the examples as well to keep things consistent.
```suggestion
        - image: k8s.gcr.io/autoscaling/cluster-autoscaler:v1.21.0
```

---

Fixed 🙂